.github/workflows/ci.yml

```diff
@@ -82,6 +82,8 @@ jobs:
     name: "cargo clippy | windows"
     steps:
       - uses: actions/checkout@v4
+      - name: "Install nasm"
+        uses: ilammy/setup-nasm@v1
```

Comment:
@zanieb I'm not very familiar with maturin, do you think installing NASM is needed for the `build-binaries` Windows job too?

https://github.com/astral-sh/uv/blob/66a4b8e6b7737b6dca78f6b60720488d0bc0889f/.github/workflows/build-binaries.yml#L153-L162

Reply:
cc @konstin

Reply:
maturin calls `cargo build`, so they same rules should apply whether it's maturin or not.